### PR TITLE
Create api end point

### PR DESCRIPTION
**Pull request by LarryIVC**
### _In this pull request, I did the following:_

- [ ] Added a branch.
- [ ] Added a model Message.
- [ ] Added a function that returns a random greeting.
- [ ] Create an API endpoint.

This API is consumed by a front-end app, the pull request link is the next:
https://github.com/LarryIVC/hello-react-front-end/pull/2

### CODE REVIEW

File: app/models/message.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+class Message < ApplicationRecord
+end
```
Comment (jerryowusu) [inline app/models/message.rb]: It is normally not a good practice to leave your model files empty. You can consider adding custom scopes, default values, associations or validations in here for better control. For instance to validate the message and make sure it is not empty you can add the line below:  Kindly check [this link](https://rails.devcamp.com/trails/learn-ruby-on-rails-from-scratch/campsites/rails-models/guides/purpose-of-model-files-in-rails) for more information. :pray: 

           validates :message, presence: true, length: { in: 1..200 }

**Review by jerryowusu — CHANGES_REQUESTED** [review]
Hi @LarryIVC , 

Good job so far!
Your project is looking fantastic so far :star_struck: 

However, there are few changes you need to make in order to move on to the next level :sunglasses:

almost got it


## Highlights
Created controller. :+1: 
Created routes :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/models/message.rb (hunk 1 above)
Comment (LarryIVC) [reply app/models/message.rb]: Hi @jerryowusu, Thanks for your feedback, I added your suggested change:

image

**Review by FrederickMih — APPROVED** [review]
## Hi :raising_hand: :raising_hand: @LarryIVC, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 

:tada: :champagne:  :tada: :champagne: 

## Highlights :green_circle: 

- Nice job adding the requested validation :+1: 
- Two apps connect together :+1: 

## Optional suggestions

- [Optional] You missed adding **Future Features** header in your README file. This header is important because it let you knows that there's always room for improvement. It is recommended to always add it in all your projects.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**